src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/index.ts

```diff
@@ -49,10 +49,11 @@ export interface IAltinnWindow extends Window {
 }
 
 export enum ProcessSteps {
-  Unknown = 0,
-  FormFilling = 1,
-  Archived = 2,
-  Confirm = 3,
+  Unknown = 'unknown',
```

Comment:
Just to make sure, have the return type here changed?

Reply:
The type that we get from the api is a string, which we then convert to our own type (this enum). So it’s easier if they match, then we don’t need to do the conversion, we can just cast it.